fr/orm/entities.rst

```diff
@@ -59,12 +59,12 @@ les données que vous voulez y stocker::
 
     $article = new Article([
         'id' => 1,
-        'title' => 'New Article',
+        'titre' => 'Nouvel Article',
```

Comment:
I don't think that we should localize field names. Content, sure, but field names feel like part of the code, even though their not real properties.

Reply:
Yes, it's a question.
I began to do it because, as a french reader, the difference between properties that are part of the code and field names that are choosen as example is not everytime clear, especially if you don't know the english vocabulary used (some rare times, luckily).
For example, when you speak about online shopping, the english word "order" can refer as well to the buying action or to a sorting action. In french, it refers only to sorting. So, keeping the english word to refer to buying action in an example leads french readers into misunderstanding.
Moreover, if the french translation is used by people who don't read easily english, having french examples makes the doc far more easy to read.
I think that a translation should look as natural as possible for foreign users, as far as it doesn't alter the original meaning.

Reply:
PS : Also, the vocabulary used in the code should match the vocabulary used in the text that describes the situation of the example. Since the text is localized, the code can/should be reasonabily localized too, in my opinion.

Reply:
Well, I guess for me it's a question of "convenience" vs industry standards. My hesitance stems from my experience with companies I've worked for where this wouldn't be accepted, as code as well as data schemas always had to be in English in order to be accessible for developers from all over the place.

So personally I'd rather see being promoted what's more common in "real world" usage, but I guess it's arguable whether this is important for the docs, so I'm not going to object if others too feel its better to localize this.

Reply:
We should keep the field names the same.

Reply:
OK guys, so just let me the time to review this in my PR